weasyl/controllers/routes.py

```diff
@@ -307,12 +306,16 @@
         'POST': director.directorcontrol_globaltagrestrictions_post_,
     }),
 
-    Route("/site-updates/", "site_update_list", general.site_update_list_),
+    Route("/site-updates/", "site_update_list", siteupdate.site_update_list_),
+    Route("/site-updates/new", "site_update_new", {
+        'GET': siteupdate.site_update_new_get_,
+        'POST': siteupdate.site_update_new_post_,
```

Comment:
This is very minor, and it’s not like we strictly adhere to a REST-style structure at all here, but I prefer `POST /site-updates/`. Future new routes will use that structure. (`site_update_put_` was also named that way because it’s logically a PUT even though `<form>`s can’t do PUT.)